docs/how-to-guides/deploy-a-flow/distribute-flow-as-executable-app.md

```diff
@@ -0,0 +1,251 @@
+# Distribute flow as executable app
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+We are going to use the [web-classification](https://github.com/microsoft/promptflow/tree/main/examples/flows/standard/web-classification/) as
+an example to show how to distribute flow as executable app with [Pyinstaller](https://pyinstaller.org/en/stable/requirements.html#).
+
+
+Please ensure that you have installed all the required dependencies. You can refer to the "Prerequisites" section in the README of the [web-classification](https://github.com/microsoft/promptflow/tree/main/examples/flows/standard/web-classification/) for a comprehensive list of prerequisites and installation instructions. And we recommend you to add a `requirements.txt` to indicate all the required dependencies for each flow. 
+
+We use [Pyinstaller](https://pyinstaller.org/en/stable/installation.html) to package the flow and [streamlit](https://docs.streamlit.io/library/get-started) to create custom web apps. So, Please install them first before distributing the flow.
```

Comment:
Briefly introduce `pyinstaller` and `streamlit` and what role they play when distributing the flow?

Reply:
updated